src/org/openvv/js/OVVAsset.js

```diff
@@ -1884,8 +1884,15 @@ Function.prototype.memoize = function() {
         return fn.memoized.apply(fn, arguments);
     }
 };
+var newOVV = new OVV();
 // initialize the OVV object if it doesn't exist
-window.$ovv = window.$ovv || new OVV();
+window.$ovv = window.$ovv || newOVV;
+for(var i in newOVV)
```

Comment:
Isn't it really dangerous to just merge these objects, regardless of their version?